editor/editor_inspector.cpp

```diff
@@ -2735,7 +2735,12 @@ void EditorInspector::update_tree() {
 
 				String label = component;
 				if (capitalize_paths) {
-					label = EditorPropertyNameProcessor::get_singleton()->process_name(label);
+					// Only process group label if this is not the group or subgroup.
+					if (!(i == 0 && label == group) || (i == 1 && label == subgroup)) {
+						label = EditorPropertyNameProcessor::get_singleton()->process_name(label);
+					} else if (EDITOR_GET("interface/editor/translate_properties")) {
+						label = TTRGET(label);
+					}
```

Comment:
The same (but without `TTRGET`, see the difference in the `make_tooltip_for_name`) probably should be done to the tooltip (set on line 2749).

Reply:
Ok, this is fixed now.